english/explore-outside/journey-to-flaming-tower/graham-cracker-cab/agree-to-pay/agree-to-pay.md

```diff
@@ -1,8 +1,26 @@
-Jumping in the cab you pay the greedy cab driver and he speeds you toward the burning tower.
+Jumping in the greedy cab driver speeds you toward the burning tower.
 
-Upon arriving at the tower you leep from the cab. With hope and courage,
+Looking through your pockets you suddendly realize you don't have near enough to cover this ride.
+
+You think about bailing out now, the ground here all appears to be based off of some strange
+glucose structure, with a good tuck-and-roll you should be fine. Your looking out the window preparing
+for your escape when you realize, your a huge p#### err... scaredy cat.
+
+It's to late now your rolling up to the burning tower. You know you have to get away this marshmellow as
+soon as he finds out you don't have his money. Then it hits you, THE FIRE... marshmellow melt!!
+
+As the vechical makes the final approach you see the firefighters all dressed in their purple 
+flame retardent suits, and just your luck there's an extra one sitting on the back of their truck.
+
+The car stops and you throw the door open. Running full speed towards the firefighters you feel yourself
+get hit in the back of the head with a heart, then star, now a horseshoe, clover, and baloon.
+
+"What the..." you scream, turning back you see a small man in a green suit riding a rainbow. He holds out 
+his hand chanting "Follow me! It's majically delicious"
 
 Do you:
 
-[Grab your purple flame retardant suit?](./grab-flame-retardant-suit/grab-flame-retardant-suit.md)
+[Keep running and grab your purple flame retardant suit?](./grab-flame-retardant-suit/grab-flame-retardant-suit.md)
+
 
+[Follow miniature carrot top, he has to have a better plan than "run into a raging inferno"](./Ride-with-Luckey/Ride-with-Luckey.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.